BusinessTampereTrafficMonitoring/traffic_lights/api_client.py

```diff
@@ -29,6 +28,15 @@ class TrafficLightCycle(Base):
 
 class TrafficLightAPIClient:
     def __init__(self, url: str, monitored_devices: List[str], db: str):
+        """
+        Constructor for initializing TrafficLightAPIClient class
+
+        #Parameter:
+        # argument1: url (String)
+        # argument2: list of monitored devices List[string]
+        # argument3: database connection string (String)
+        """
```

Comment:
You should use the same convention in all of the docstrings. In other files you said the name of the parameter rather than "argument1", "argument2" (which I think makes more sense). Stick to that for all of the docstrings.